rhaptos/cnxmlutils/xsl/cnxml-to-html5.xsl

```diff
@@ -131,6 +131,16 @@
   </math>
 </xsl:template>
 
+<!-- ========================= -->
+<!-- Math Sanitization         -->
+<!-- ========================= -->
+
+<xsl:template match="m:msubsup[count(*)=2]">
```

Comment:
Would not it be safer to use `m:msubsup[count(*) &lt; 3]` ?

Reply:
Good point. I added a template that accounts for < 2 elements (just remove the `<m:msubsup>`)

Reply:
:+1: